management/sites/sites.php

```diff
@@ -0,0 +1,318 @@
+			<ul class="unstyled-list wdn-list-reset">
+				<li class="site-list-element"
+					ng-repeat="site in sites"
+					ng-click="selectSite(site)">
+					<a>{{site.title}}</a>
+				</li>
+			</ul>
+
+			<!-- Shown when there are no sites -->
+			<div ng-if="sites == null" class="centered-text">
+				<p>There are no sites.</p>
+			</div>
+		</div>
+
+
+
+
+
+		<!-- Site Information Area -->
+		<div class="wdn-col-three-fourths">
+			<!-- Shown if a site is selected and loaded from backend -->
+			<div ng-if="selectedSite != null && siteInformation != null">
+				<h3 class="clear-top">{{siteInformation.title}}</h3>
+				<div class="wdn-grid-set-halves">
+					<div class="wdn-col">
+						<div><b>Address:</b> {{siteInformation.address}}</div>
+						<div><b>Phone Number:</b> {{siteInformation.phoneNumber}}</div>
+					</div>
+					<div class="wdn-col">
+						<div><b>Does Multilingual:</b> {{siteInformation.doesMultilingual ? 'Yes' : 'No'}}</div>
+						<div><b>Does International:</b> {{siteInformation.doesInternational ? 'Yes' : 'No'}}</div>
+					</div>
+				</div>
+
+
+				<!-- Volunteer Shifts Section -->
+				<h4>Volunteer Shifts:</h4>
+				<div>
+					<!-- Displayed when there are no volunteer shifts -->
+					<div ng-if="siteInformation.shifts == undefined || siteInformation.shifts.length <= 0">
+						<p class="clear-top">There are no shifts</p>
+					</div>
+
+					<!-- Displayed when there are volunteer shifts -->
+					<div ng-if="siteInformation.shifts != undefined && siteInformation.shifts.length > 0">
+						<table class="wdn_responsive_table">
+							<thead>
+								<tr>
+									<th id="shiftDateHeader">Date</th>
+									<th id="shiftStartTimeHeader">Start Time</th>
+									<th id="shiftEndTimeHeader">End Time</th>
+								</tr>
+							</thead>
+							<tbody>
+								<tr ng-repeat="shift in siteInformation.shifts">
+									<th id="shift{{shift.shiftId}}">{{shift.dateString}}</div>
+									<td headers="shiftStartTimeHeader shift{{shift.shiftId}}">{{shift.startTimeString}}</td>
+									<td headers="shiftEndTimeHeader shift{{shift.shiftId}}">{{shift.endTimeString}}</td>
+								</tr>
+							</tbody>
+						</table>
+					</div>
+
+					<!-- Add volunteer shift section -->
+					<div ng-if="addShiftButtonClicked === true">
+						<h5>Add a Shift</h5>
+						<form class="cmxform"
+							id="addShiftForm"
+							name="form"
+							ng-submit="form.$valid && addShiftSaveButtonHandler()" 
+							autocomplete="off" 
+							novalidate>
+
+							<div class="form-textfield">
+								<label class="form-label form-required">Date</label>
+								<input type="text" 
+									id="addShiftDateInput" 
+									name="dateInput" 
+									placeholder=" -- Select a Date -- " 
+									autocomplete="off"
+									required>
+								<div ng-show="form.$submitted || form.dateInput.$touched">
+									<label class="error" ng-show="form.dateInput.$error.required">This field is required.</label>
```

Comment:
FYI, although you can't select a prior date, you can still change the text to a prior date (and it does create a new shift on that past date without any errors). Not necessarily a bug unless we really don't want them to create new shifts on past dates. Shouldn't create any problems though if they do for some odd reason.

Reply:
I'm partial to letting this one slide as is.

Reply:
Sounds good.